### PR TITLE
1-4 Task Dmitry Gaiduk

### PR DESCRIPTION
**Pull request by IdokkeI**
https://travis-ci.com/github/IdokkeI/js-assignments/builds/177822540

### CODE REVIEW

**Comment by FedorovD** [comment]
Provide travis link pls and remove all `// throw new Error('Not implemented');`  lines

**Comment by IdokkeI** [comment]
https://travis-ci.com/github/IdokkeI/js-assignments/branches

File: task/01-strings-tasks.js [hunk 1]
```diff
@@ -69,7 +69,8 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
 function extractNameFromTemplate(value) {
-    throw new Error('Not implemented');
+    
+    return value.slice(7, value.length-1);
```
Comment (FedorovD) [inline task/01-strings-tasks.js]: looks good, but please note slice can work with negative numbers also to take symbols from end. In the case `slice(7, -1);` can do the same.

Comment (IdokkeI) [reply task/01-strings-tasks.js]: Ok

Comment (IdokkeI) [reply task/01-strings-tasks.js]: I'll need to fix that?

**Comment by IdokkeI** [comment]
link to new pr: https://github.com/AisBrestEDU/js-assignments/pull/950 reset master and create new pr for check.
link to travic: https://travis-ci.com/github/IdokkeI/js-assignments/builds/181789567

**Comment by vramaniuk** [comment]
I'm closing this, because we have the new one

**Pull request closed without merging**